Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -1339,4 +1344,19 @@ std::set<std::string> HoI4::World::getSouthAsianCountries() const
 	}
 
 	return southAsianCountries;
+}
+
+
+void HoI4::World::addProvincesToHomeAreas()
+{
+	Log(LogLevel::Info) << "Adding provinces to home areas";
+	for (const auto& country: landedCountries | std::views::values)
+	{
+		const auto& capital = country->getCapitalProvince();
+		if (!capital)
+		{
+			continue;
+		}
+		country->addProvincesToHomeArea(*capital, theMapData, getStates(), getProvinceToStateIDMap());
```

Comment:
```suggestion
		country->addProvincesToHomeArea(*capital, theMapData, states->getStates(), states->getProvinceToStateIDMap());
```

Reply:
These are alias functions, class members

Reply:
I have a mild preference to not use the public interface when possible, but if you have a preference otherwise feel free to just mark this as resolved and move on.